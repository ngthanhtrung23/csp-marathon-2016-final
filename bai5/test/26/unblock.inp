1112.3
4452.3
005..6
788996
7.AABB
CCC...
