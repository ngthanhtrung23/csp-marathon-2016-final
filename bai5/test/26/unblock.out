27
A L 1
B L 1
6 D 2
3 D 1
9 R 1
2 D 2
1 R 3
C R 1
7 D 1
8 L 1
5 D 1
4 R 3
5 U 2
0 R 1
8 R 1
7 U 4
0 L 1
8 L 1
A L 1
5 D 3
0 R 1
4 L 2
7 D 1
1 L 3
2 U 2
3 U 1
0 R 3

